app/views/portal/show.rb

```diff
@@ -37,7 +37,8 @@ def head_meta
 
     def page_title
       mustache[:page_title] ||= begin
-        CGI.unescapeHTML([document.fetch(:title, ['']).join(', '), 'Europeana'].compact.join(' - '))
+        title = [render_document_show_field_value(document, 'proxies.dcTitle', unescape: true), creator_title]
+        title = CGI.unescapeHTML(title.compact.join(' | '))  + ' - Europeana'
```

Comment:
Useless assignment to variable - `title`.<br>Unnecessary spacing detected.<br>Operator `+` should be surrounded with a single space.